src/components/CtaCard/CtaCard.js

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+
+import "./CtaCard.css";
+
+const CtaCard = ({ children, title, iconHandle, link }) => {
+  return (
+    <div className="CtaCard">
+      <a href={link} title={title}>
+        <div className="CtaCard__heading">
+          <div className={`CtaCard__icon CtaCard__icon--${iconHandle}`}>
+
+          </div>
+          <h2 className="CtaCard__title">
+            {title}
+            </h2>
+        </div>
+        <div className="CtaCard__body">
+          {children}
+          </div>
+      </a>
+    </div>
+  );
+};
+
+CtaCard.propTypes = {
+  children: PropTypes.node,
+  title: PropTypes.node,
```

Comment:
Are You sure that this is node (html element) ? Maybe it's string. If label could be node write something like this
label: PropTypes.oneOfType([ PropTypes.string, PropTypes.node ]),